sandwich/src/index.ts

```diff
@@ -53,14 +54,13 @@ function provideHandleTransaction(
     // extract all swap events
     for (let i in txs) {
       const data = await web3.eth.getTransaction(txs[i]);
-
-      const decodedData = abiDecoder.decodeMethod(data.input);
+      let ID = (data.input).slice(0,10);
 
       if (
-        data.to === contractAddress &&
-        decodedData != undefined &&
-        decodedData.name === "swapTokensForExactTokens"
+        Method_ID.includes(ID) &&
```

Comment:
I would prefer to here an equality check and get rid of the array because there is no need to use `array.includes` when the array contains only one element.